### PR TITLE
IGNITE-9010: Move web-console build to dedicated root directory

### PR DESCRIPTION
**Pull request by imurchenko**

### CODE REVIEW

File: docker/web-console/standalone/Dockerfile [hunk 1]
```diff
@@ -15,62 +15,76 @@
...
+# Install global node packages
 RUN npm install -g pm2
 
-WORKDIR /opt/web-console
-
-COPY docker/standalone/docker-entrypoint.sh docker-entrypoint.sh
-
-RUN chmod +x docker-entrypoint.sh \
-    && dos2unix docker-entrypoint.sh
+# Update software sources and install missing applications
```
Comment (nva) [inline docker/web-console/standalone/Dockerfile]: May be moving this code block is not good idea (this hide real changes and increase diff)?

File: docker/web-console/standalone/Dockerfile [hunk 2]
```diff
@@ -15,62 +15,76 @@
...
+# Copy and fix entrypoint script
+COPY docker-entrypoint.sh docker-entrypoint.sh
+RUN chmod +x docker-entrypoint.sh \
+    && dos2unix docker-entrypoint.sh
+
+# Copy nginx configuration
+COPY nginx/* /etc/nginx/
+
+# Mount required volumes
 VOLUME ["/etc/nginx"]
```
Comment (nva) [inline docker/web-console/standalone/Dockerfile]: Is this volumes really needed?

File: docker/web-console/standalone/README.txt [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+==================================================
+Apache Ignite Web Console Standalone Docker module provides Dockerfile and accompanying files
+for building docker image of Web Console.
+
+
+Ignite Web Console Standalone Docker Image Build Instructions
+=============================================================
+1) Build ignite-web-console module
+
+        mvn clean install -P web-console -DskipTests -T 2C -pl :ignite-web-console -am
```
Comment (nva) [inline docker/web-console/standalone/README.txt]: Is `-T 2C` arguments really required?

File: docker/web-console/standalone/Dockerfile (hunk 1 above)
Comment (vveider) [reply docker/web-console/standalone/Dockerfile]: I did not quite understand what was comment about, but I've merged 2 RUN commands into 1 for less layers number and better control over software installation process.

File: docker/web-console/standalone/Dockerfile (hunk 2 above)
Comment (vveider) [reply docker/web-console/standalone/Dockerfile]: @imurchenko -- can you comment on this?

File: docker/web-console/standalone/README.txt (hunk 3 above)
Comment (vveider) [reply docker/web-console/standalone/README.txt]: Not really, but that increases build speed by 15-25%, especially on multi-core systems

**Pull request closed without merging**